resources/multisite_drupal_standard.make

```diff
@@ -528,6 +534,10 @@ projects[og][patch][] = patches/og-og_field_access-bypass_field_access-5159.patc
 ; https://www.drupal.org/node/1880226
 projects[og][patch][] = https://www.drupal.org/files/issues/og-use_numeric_id_for_membership_etid-1880226-5.patch
 
+; NEXTEUROPA-13743 Issue with og_field_access hiding all content fields on creation
+; https://www.drupal.org/node/1743332
+projects[og][patch][] = https://www.drupal.org/files/issues/og-hidden-body-field-1743332-2_0.patch
```

Comment:
This patch fails in d.org.

Reply:
But this patch is needed... what you propose ?

Reply:
What's the ticket reference for this patch @gervasek ?

Reply:
https://webgate.ec.europa.eu/CITnet/jira/browse/NEPCB-19

Reply:
The patch needs work, the dev should work on this patch before we can include it.

Reply:
Step to reproduce are in the NEPCB-19 ticket:
```
Steps to reproduce :

    install the platform
    activate the og_field_access module
    create a user with content creation rights ( for exemple editor )
    go to a content create form ( for exemple article )
    there should be no field, except title
```